package-lock.json

```diff
@@ -2603,6 +2634,73 @@
         "npm": "^7"
       }
     },
+    "node_modules/@readme/oas-to-har": {
+      "version": "13.6.1",
+      "resolved": "https://registry.npmjs.org/@readme/oas-to-har/-/oas-to-har-13.6.1.tgz",
+      "integrity": "sha512-af56xwomcTw//LyXSu67sNcYTMwptjEITrbySDHltFrbhRx6gY/1zvniXJJm4nrQ/yam9vqCoH+dIOwpjbbSuQ==",
+      "dev": true,
+      "dependencies": {
+        "@readme/oas-extensions": "^13.6.0",
+        "oas": "^14.0.0",
+        "parse-data-url": "^4.0.1"
+      },
+      "engines": {
+        "node": "^12 || ^14 || ^16",
+        "npm": "^7"
+      }
+    },
+    "node_modules/@readme/oas-to-snippet": {
+      "version": "13.6.2",
+      "resolved": "https://registry.npmjs.org/@readme/oas-to-snippet/-/oas-to-snippet-13.6.2.tgz",
+      "integrity": "sha512-1X+LsPA+hT6pO6GNeq2tdXjv4O1Q316XPmpQVg+zv5e65fY/Dk6m+7Scxf0eAWkvUf0ZRU32d+4XBj2HNh5Aew==",
+      "dev": true,
+      "dependencies": {
+        "@readme/httpsnippet": "^2.5.1",
+        "@readme/oas-extensions": "^13.6.0",
+        "@readme/oas-to-har": "^13.6.0",
+        "@readme/syntax-highlighter": "^10.10.1",
+        "httpsnippet-client-api": "^3.3.0",
+        "oas": "^14.0.0"
+      },
+      "engines": {
+        "node": "^12 || ^14 || ^16",
+        "npm": "^7"
+      }
+    },
+    "node_modules/@readme/syntax-highlighter": {
```

Comment:
I hate that `oas-to-snippet` loads `syntax-highlighter` as a dependency just for this file https://github.com/readmeio/syntax-highlighter/blob/main/src/utils/uppercase.js

Reply:
Refactored `oas-to-snippet` to remove the `syntax-highlighter` dep and published a breaking change v14.0.0 release.

https://github.com/readmeio/oas-to-snippet/blob/main/CHANGELOG.md#1400-2021-09-02